docs/Policy_for_OBO_namespace_and_associated_PURL_requests.md

```diff
@@ -18,38 +23,36 @@ The technical working group reserves the right to make decisions based on their
+  1. We recommend projects be orthogonal to each other (i.e. no domain overlap). For those classes that are common across multiple resources (e.g., the class _cell_) we recommend importing terms from an already existing ontology. Application ontologies (those developed for specific data integration tasks) should reuse terms from existing OBO Foundry domain ontologies. At a minimum, we expect attribution of existing work, specifically through the use of PURLs for individual terms and ontologies.
 
 ## Status of ontology development ##
 
   1. A project should exist, with work started. We will not "pre-book" prefixes and domains for potential future resources.
+  1. A project must be doing original work within the biological ontology community. 
+      * We will not provide namespaces for projects that are issuing new identifier for existing ontologies.
+      * If there is clear overlap with an existing domain ontology, you must demonstrate a good faith effort to work with that ontology to provide any new terms needed.
   1. The required namespace must be available.
-  1. The resource must be publicly available when released.
-  1. There must be a contact person for the resource. We ask that the contact person for resources be subscribed to our main communication channel, the obo-discuss list.
+  1. The resource must be publicly available when released. The general expectation is that the ontology source code is available on a public repository such as GitHub.
+  1. There must be a contact person for the resource. The contact person for resources must be subscribed to our main communication channel, the obo-discuss list.
   1. The requestor and/or contact person should be ready to discuss issues such as whether the ontology is orthogonal, whether there is potential to collaborate with existing efforts.
-  1. It is expected that solicitation of a prefix is done _before_ the prefix is used for identifiers. A common strategy is to develop an ontology, request a prefix, and translate the initial URIs used to the PURLs some time before the initial release. **There is no guarantee that you will be granted your prefix, even if you have been using it in your file**.
+  1. It is expected that solicitation of a prefix is done _before_ the prefix is used for identifiers. A common strategy is to develop an ontology, request a prefix, and translate the initial URIs used to the PURLs some time before the initial release.  **There is no guarantee that you will be granted your prefix, even if you have been using it in your file**.
 
-## Information needed for a prefix request ##
 
-You must first make your ontology available via a stable URL. Provide this to us, together with the following metadata:
+## Requesting the prefix ##
+
+  1. First send an email to [obo-discuss](mailto:obo-discuss@googlegroups.com) with your request to allow community feedback (you may need to [apply to join group](https://groups.google.com/forum/#!forum/obo-discuss) first.)
```

Comment:
Why not have them first submit the issue, then the issue can be referenced in the email?

Reply:
Great idea!

Reply:
The policy predates our use of github. Time to bring it up to date.